.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: package-plugin-workflow
+
+on: [push]
+
+jobs:
+  package-plugin-job:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        unreal: [ue-4-27, ue-5-0]
+        include:
+        - unreal: ue-4-27
+          UE_IMAGE: "ghcr.io/epicgames/unreal-engine:dev-slim-4.27"
+        - unreal: ue-5-0
+          UE_IMAGE: "ghcr.io/epicgames/unreal-engine:dev-slim-5.0.2"
+
+    container:
+      image: ${{ matrix.UE_IMAGE }}
+      credentials:
+        username: ${{ secrets.UE_REPO_USER }}
+        password: ${{ secrets.UE_REPO_ACCESS_TOKEN }}
+      options: --user root
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Installing Linux Dependencies
+        run: |
+          sudo apt-get update
+          pip3 install --upgrade pip
```

Comment:
just in case we needed to install more stuff, it may be worthwhile to instead run the the container inside the GH action, as opposed to running the GH action in the container. Like this https://github.com/getsentry/sentry-unity/blob/76f0e16886974effca3f0df4aad03876b30d0d0b/.github/workflows/ci.yml#L84

Reply:
Not sure the ue4cli would work with that though. also, is it worth installing just for a single call (`ue4 package`)?

Reply:
I've tried running the container inside the GH action in this [commit](https://github.com/getsentry/sentry-unreal/pull/29/commits/7f487849a66e7878cf566d36377445492fb23c07), though the current approach seems to look a bit cleaner to me. Not sure if that is going to make any difference atm, since both solutions work identically

Reply:
sounds good